.tekton/hypershift-cli-mce-27-pull-request.yaml

```
@@ -37,575 +37,15 @@ spec:
-      taskRef:
-        params:
-        - name: name
-          value: init
-        - name: bundle
-          value: quay.io/konflux-ci/tekton-catalog/task-init:0.2@sha256:66e90d31e1386bf516fb548cd3e3f0082b5d0234b8b90dbf9e0d4684b70dbe1a
-        - name: kind
-          value: task
-        resolver: bundles
-    - name: clone-repository
-      params:
       - name: url
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)